www/pages/blog/release/v0-28-0.md

```diff
@@ -0,0 +1,136 @@
+  constructor(compilation, options) {
+    super(compilation, options);
+    this.extensions = ['.css'];
+    this.contentType = 'text/css';
+  }
+
+  async serve(url) {
+    const body = await fs.promises.readFile(url, 'utf-8');
+
+    return {
+      body,
+      contentType: this.contentType
+    };
+  }
+}
+```
+
+And here is what it looks like now, exclusively based on Web APIs.  Nothing ad-hoc anymore! ✨
+```js
+// eslint-disable-next-line no-unused-vars
+class StandardCssResource extends ResourceInterface {
+  constructor(compilation, options) {
+    super(compilation, options);
+    this.extensions = ['css'];
+    this.contentType = 'text/css';
+  }
+
+  // defining shouldServe is now required
+  async shouldServe(url) {
+    const { protocol, pathname } = url;
+
+    return protocol === 'file:' && this.extensions.includes(pathname.split('.').pop());
+  }
+
+  async serve(url) {
+    const body = await fs.promises.readFile(url, 'utf-8');
+
+    return new Response(body, {
+      headers: {
+        'Content-Type': this.contentType
+      }
+    });
+  }
+}
+```
+
+> _This refactor touched most of our plugin APIs, so you'll want to read our [release notes](https://github.com/ProjectEvergreen/greenwood/releases/tag/v0.28.0) to learn about how to migrate any of your own custom plugins._
+
+### API Routes
+
+To fully round out Greenwood's server rendering capabilities, API Routes are now available.  This API is based on a simple `Request` / `Response` model leveraging Web APIs directly as inputs and outputs.  Let's take a look at an example.
+
+API routes follow a file based routing convention, just like [pages](/docs/layouts/#pages).  So this structure will yield an endpoint available at `/api/greeting`.
+```shell
+src/
+  api/
+    greeting.js
+```
+
+This example API reads a query parameter of `name` and returns a JSON response based on that.
+
+```js
+export async function handler(request) {
+  const params = new URLSearchParams(request.url.slice(request.url.indexOf('?')));
+  const name = params.has('name') ? params.get('name') : 'World';
+  const body = { message: `Hello ${name}! 👋` };
+
+  return new Response(JSON.stringify(body), {
+    headers: new Headers({
+      'Content-Type': 'application/json'
+    })
+  });
+}
+```
+
+All web standards, all the time.  You know we love to see it. 🤓
+
+## What's Next
+
+In addition to the above mentioned features, this release also lays the ground work for our foray into [Serverless and Edge runtime support](https://github.com/ProjectEvergreen/greenwood/issues/1008).  Anticipated for our next release, Greenwood will be able to fully embrace running beyond the server with hosting providers like Netlify and Vercel all while benefiting from the capabilities of fully embracing web standards. We [see this trend becoming more ubiquitous](https://wintercg.org/) as more and more hosting providers coalesce around these standards too, so now is a great time to get started!
```

Comment:
hi @thescientist13, I didn't notice any other spelling mistakes but perhaps we can improve 134 to become less wordy:

"In anticipation of our next release, Greenwood will be able to fully embrace running beyond the server with hosting providers such as Netlify and Vercel, all while benefiting from the capabilities of fully adhering to web standards"

Reply:
Hmm, yeah, I would see tweaking this a bit.  Gotta pump up that FOMO!